README.md

```diff
@@ -27,7 +27,7 @@ After you're finished please remove all the comments and instructions!
 
 <div align="center">
   <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+ 
   <br/>
 
   <h3><b>Microverse README Template</b></h3>
```

Comment:
- [ ] [OPTIONAL] Please remove this heading, as we only need to use the template for our projects and do not have to highlight it as a template